### PR TITLE
CONCF-1063 | Style section template

### PR DESCRIPTION
**Pull request by vforge**

### CODE REVIEW

File: front/styles/main/component/_curated-content-editor.scss [hunk 1]
```diff
@@ -119,55 +118,42 @@ $search-results-load-more-button-width: 100vw - 2 * $search-results-horizontal-p
...
-		svg {
-			fill: $color-blue-dark;
-			height: 1.75rem;
-			width: 1.75rem;
-		}
-	}
+	color: $color-blue-brand;
+	font-size: 1.1rem;
+	font-weight: bold;
+	line-height: 1.437rem;
```
Comment (vforge) [inline front/styles/main/component/_curated-content-editor.scss]: 23px

File: front/styles/main/component/_curated-content-editor.scss [hunk 2]
```diff
@@ -119,55 +118,42 @@ $search-results-load-more-button-width: 100vw - 2 * $search-results-horizontal-p
...
+}
+.curated-content-section-details--count {
+	color: $color-blue-gray-dark;
+	font-size: .75rem;
+	line-height: .875rem;
```
Comment (vforge) [inline front/styles/main/component/_curated-content-editor.scss]: 14px

File: front/styles/main/component/_curated-content-editor.scss (hunk 1 above)
Comment (hakubo) [inline front/styles/main/component/_curated-content-editor.scss]: how about 1.4rem?

File: front/styles/main/component/_curated-content-editor.scss [hunk 3]
```diff
@@ -119,55 +118,42 @@ $search-results-load-more-button-width: 100vw - 2 * $search-results-horizontal-p
...
+}
 
-	.section-items-count {
-		color: $color-blue-gray-dark;
-		font-size: .75rem;
-		line-height: .9rem;
-		padding: 0 13px 10px;
+.curated-content-section-details--description {
+	flex: 1;
+	margin: {
```
Comment (hakubo) [inline front/styles/main/component/_curated-content-editor.scss]: margin shorthand FTW

File: front/styles/main/component/_curated-content-editor.scss (hunk 1 above)
Comment (vforge) [inline front/styles/main/component/_curated-content-editor.scss]: It's 22,4px, rounded to 22px :(

Comment (hakubo) [inline front/styles/main/component/_curated-content-editor.scss]: then 1,5rem - 24px

Let's try not to use such magic numbers

Comment (vforge) [inline front/styles/main/component/_curated-content-editor.scss]: But that's one reason to use `rem-calc` ;)

**Comment by vforge** [comment]
I'll wait for 989 to be merged to dev before solving conflicts.

**Pull request closed without merging**